octopusdeploy/resource_runbook_process.go

```diff
@@ -72,6 +72,7 @@ func resourceRunbookProcessCreate(ctx context.Context, d *schema.ResourceData, m
 
 	runbookProcess.ID = current.ID
 	runbookProcess.Links = current.Links
+	runbookProcess.ProjectID = runbook.ProjectID
```

Comment:
The ProjectID is required now in later Octopus Server versions. This will be particularly important when we support CaC runbooks